src/awscr-s3/presigned/form.cr

```diff
@@ -24,8 +24,17 @@ module Awscr
         #   form.condition("success_action_status", "201")
         # end
         # ```
-        def self.build(region, aws_access_key, aws_secret_key, signer = :v4, &block)
-          post = Post.new(region, aws_access_key, aws_secret_key, signer)
+        def self.build(region : String, aws_access_key : String, aws_secret_key : String, aws_session_key : String? = nil, signer = :v4, &block)
```

Comment:
I see that you mentioned the specified type for `signer` in the Deprecated message.

```suggestion
        def self.build(region : String, aws_access_key : String, aws_secret_key : String, aws_session_key : String? = nil, signer : Symbol = :v4, &block)
```

Reply:
Thanks for picking up the project!

(For the change) I would not mind to add the type, but for understanding: is that not implied by the default (`:v4`), which is a Symbol? Or is it more for readability to explicitly list the type?

Reply:
I think it's fine with or without `type`, but shouldn't the line [[here](https://github.com/taylorfinnell/awscr-s3/pull/115/files#diff-fd66474d3de243160eaf256bae902da646ebce170904a0601647ee32690fa273R35)](https://github.com/taylorfinnell/awscr-s3/pull/115/files#diff-fd66474d3de243160eaf256bae902da646ebce170904a0601647ee32690fa273R35) have the same signature for consistency?